docs/api.md

```diff
@@ -2931,9 +2930,9 @@ function, it **will not be called**.
 
 #### consoleMessage.location()
 - returns: <[Object]>
```

Comment:
We return an object with optional fields? Let's make them nullable instead.

Reply:
These are helpful when JSON stringifiying or console logging these objects in node. I'd rather not change implementation in this patch.